nextflow.config

```diff
@@ -88,48 +90,26 @@ params {
     monochrome_logs              = false
     hook_url                     = null
     help                         = false
+    help_full                    = false
+    show_hidden                  = false
     version                      = false
     pipelines_testdata_base_path = 'https://raw.githubusercontent.com/nf-core/test-datasets/'
-
-    // Config options
+    trace_report_suffix          = new java.util.Date().format( 'yyyy-MM-dd_HH-mm-ss')// Config options
```

Comment:
Just a detail but it looks like you merged the new line with the comment line by accident